src/uct/api/v2/uct_v2.h

```diff
@@ -577,6 +593,37 @@ typedef struct uct_iface_is_reachable_params {
 } uct_iface_is_reachable_params_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Operation parameters passed to @ref uct_ep_is_connected.
+ *        Only addresses with addr_len > 0 should be the passed.
+ */
+typedef struct uct_ep_is_connected_params {
+    /**
+     * Mask of valid fields in this structure, using
+     * bits from @ref uct_ep_is_connected_field_mask_t. Fields not specified
+     * in this mask will be ignored. Provides ABI compatibility with respect to
+     * adding new fields.
+     */
+    uint64_t                 field_mask;
+
+    /**
+     * Device address to check for connectivity.
```

Comment:
maybe you can add notes, like we have for  `uct_iface_is_reachable_params_t`
`This field must not be passed if iface_attr.dev_addr_len == 0.`

same is relevant for other addresses in this struct

Reply:
maybe keep it as `iface_attr.dev_addr_len`